packages/plugin-autocapture-browser/src/hierarchy.ts

```diff
@@ -0,0 +1,135 @@
+import { isNonSensitiveElement } from './helpers';
+import { Hierarchy, HierarchyNode } from './typings/autocapture';
+
+const BLOCKED_ATTRIBUTES = [
+  // Already captured elsewhere in the hierarchy object
+  'id',
+  'class',
+
+  // non-useful and potentially large attribute
+  'style',
+
+  // sensitive as prefilled form data may populate this attribute
+  'value',
+
+  // DOM events
+  'onclick',
+  'onchange',
+  'oninput',
+  'onblur',
+  'onsubmit',
+  'onfocus',
+  'onkeydown',
+  'onkeyup',
+  'onkeypress',
+
+  // React specific
```

Comment:
Should we also consider adding a allow list to combine, since other frameworks might introduce attributes too

Reply:
Yes, good point. As a follow up diff, I'll investigate options to expose for customizing data collection